#Actividad Evaluable: Obtención de estadisticas descriptivas
  TC1002

Barbara Nicole Vidal Sandoval | A01635233

Misael Octavio Rodríguez Macías | A01639786

Daniel Alejandro Flores García | A00831637


  
*   Importamos las librerías necesarias
*   Cargamos el archivo de base de datos .csv


In [ ]:
import pandas as pd
#import numpy as np
from matplotlib import pyplot as plt
#import random

df = pd.read_csv("/01.07.2022_da_hospitalizados_covid19 (1).csv")
df

Fecha      Institución  \
0                                          21/03/2020              SSJ   
1                                          21/03/2020              HCG   
2                                          21/03/2020           ISSSTE   
3                                          21/03/2020             IMSS   
4                                          21/03/2020         Privados   
...                                               ...              ...   
6266                                       01/07/2022   SMM TLAJOMULCO   
6267                                       01/07/2022  SMM GUADALAJARA   
6268                                       01/07/2022        CRUZ ROJA   
6269                                       01/07/2022     SSMM ZAPOPAN   
6270  Información actualizada al 01 de julio de 2022.              NaN   

      Pacientes  Positivos  En_estudio  
0           0.0        0.0         0.0  
1           2.0        0.0         2.0  
2           0.0        0.0         0.0  
3           0.0        0.0         0.0  
4           0.0        0.0         0.0  
...         ...        ...         ...  
6266        0.0        0.0         0.0  
6267        0.0        0.0         0.0  
6268        0.0        0.0         0.0  
6269        0.0        0.0         0.0  
6270        NaN        NaN         NaN  

[6271 rows x 5 columns]



*   Observamos la dimensión de la base de datos



In [ ]:
df.shape

(6271, 5)



*   Desplegamos información general de la base de datos, como tipo de variables



In [ ]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 6271 entries, 0 to 6270
Data columns (total 5 columns):
 #   Column       Non-Null Count  Dtype  
---  ------       --------------  -----  
 0   Fecha        6271 non-null   object 
 1   Institución  6270 non-null   object 
 2   Pacientes    6270 non-null   float64
 3   Positivos    6270 non-null   float64
 4   En_estudio   6270 non-null   float64
dtypes: float64(3), object(2)
memory usage: 245.1+ KB




*   Observamos la cantidad y el nombre de las columnas junto con el tipo de variable



In [ ]:
df.columns

Index(['Fecha', 'Institución', 'Pacientes', 'Positivos', 'En_estudio'], dtype='object')



*   Desplegamos las primeras lineas de nuestra base de datos a pantalla



In [ ]:
df.head()

Fecha Institución  Pacientes  Positivos  En_estudio
0  21/03/2020         SSJ        0.0        0.0         0.0
1  21/03/2020         HCG        2.0        0.0         2.0
2  21/03/2020      ISSSTE        0.0        0.0         0.0
3  21/03/2020        IMSS        0.0        0.0         0.0
4  21/03/2020    Privados        0.0        0.0         0.0


Por último observamos diferentes mediciones matemáticas que describen el comportamiento cuantitativo de la información dentro de la base de datos como:
*   Cantidad
*   Desviación estándar
*   Mínimo
*   Máximo
*   Media
*   Diferentes porcentajes






In [ ]:
df.describe()

Pacientes    Positivos   En_estudio
count  6270.000000  6270.000000  6270.000000
mean     71.354067    43.816427    27.507177
std     146.396509    85.259193    64.762094
min       0.000000     0.000000     0.000000
25%       0.000000     0.000000     0.000000
50%      12.000000     7.000000     3.000000
75%      84.000000    52.000000    23.000000
max    1257.000000   719.000000   583.000000

Conclusión

Después de observar los datos y los valores que nos arroja como la media y la desviación, concluimos que los casos positivos al ser la media de 43 por hospital en la región total de Jalisco, hay una gran baja de casos comparado con los anticipados en la pandemia, también vemos que aunque el mayor número de casos positivos registrados en un hospital es de 719, la mayoria de los hospitales tiene menos de 50 casos, por lo que aún asi son muy pocos casos, sobretodo en estos tiempos que el cubrebocas ya no es obligatorio.

El análisis nos menciona que cada hospital tiene una media de 43 pacientes positivos en Covid, sin embargo, esta afirmación es erronea, ya que el valor de la desviación estándar es aproximadamente el doble que la media, esto demuestra que la cantidad de pacientes y casos de covid no estan distribuidos de manera "normal", más bien estan distribuidos de manera que se pueden observar grandes agrupaciones, lo cual se puede observar en base a los mínimos y máximos de las cantidades de casos positivos.